packages/lab/src/typography/Text.css

```diff
@@ -1,10 +1,10 @@
 /* Body emphasis */
 .uitkText strong {
-  font-weight: var(--uitk-text-weight-strong);
+  font-weight: var(--uitk-text-fontWeight-strong);
```

Comment:
It's kind of confusing where text uses fontWeight and typography uses weight

Reply:
typography weights will be removed at some point